modules/doc/content/getting_started/installation/install_libtorch.md

```diff
@@ -62,9 +83,10 @@ The user can choose from two alternatives when it comes to installing `libtorch`
   [official website](https://github.com/pytorch/pytorch/blob/master/docs/libtorch.rst).
 
 
-## Configure and compile MOOSE with libtorch
+## Configure and MOOSE with libtorch
```

Comment:
Is there a way we can shorten this? I find anything longer than X characters, and the navigation table truncates.

Another suggestion would be to use a smaller heading (an additional # character).

I typically reserve `##` for short subject matter headings like "Setup MOOSE", "Configure MOOSE", etc. And then lengthy pretty headings `###` for separating sections within the main subject matter heading.

Reply:
Sounds great, modified it to `Configuring MOOSE` and `Building MOOSE`.